guilford_dps_offenders.yaml

```diff
@@ -0,0 +1,330 @@
+#psql :  offender_profile
+# intellij: "select * from offender_profile;"
+
+
+entityDefinitions:
+  offender:
+    fqn: "general.person"
+    entitySetName: "GuilfordOffenders"
+    propertyDefinitions:
+      nc.SubjectIdentification:
+        type: "nc.SubjectIdentification"
+        column: "OFFENDER NC DOC ID NUMBER"
+      nc.PersonSex:
+        type: "nc.PersonSex"
+        column: "OFFENDER GENDER CODE"
+        transforms:
+        - !<transforms.ReplaceTransform>
+          target: ["MALE", "FEMALE"]
+          goal: ["M", "F"]
+      nc.PersonBirthDate:
+        type: "nc.PersonBirthDate"
+        column: "OFFENDER BIRTH DATE"
+        transforms:
+        - !<transforms.DateTransform>
+          pattern: ["yyyy-MM-dd HH:mm:ss", "yyyy-MM-dd HH:mm:ss.S"]
+      nc.PersonRace:
+        type: "nc.PersonRace"
+        # column: "OFFENDER RACE CODE"
+        transforms:
+        - !<transforms.BooleanRegexTransform>
+          column: "OFFENDER RACE CODE"
+          pattern: "(\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?)"
+          transformsIfTrue:
+          - !<transforms.ValueTransform>
+            value: null
+          transformsIfFalse:
+          - !<transforms.ColumnTransform>
+            column: "OFFENDER RACE CODE"
+          - !<transforms.ReplaceTransform>
+            target: [ "BLACK", "UNKNOWN", "OTHER", "ASIAN/ORTL", "INDIAN", "WHITE"]
+            goal: ["black", "", "other", "asian", "amindian", "white"]
+      nc.PersonRace2:
+        type: "nc.PersonRace"
+        # column: "OFFENDER ETHNIC CODE"
```

Comment:
please delete this commented out line also.

Reply:
will do